docs/products/access/release-notes/1.x.md

```diff
@@ -1,5 +1,31 @@
 # Release notes 1.x versions
 
+## 1.1.1
+
+### Bug fixes
+
+* Resolved an issue causing some of the Access Admin's templates from being rendered correctly.
+
+## 1.1.0
+
+### Features
+
+* Mobile app developers can now modify the redirect URL that is configured in the Access Admin without breaking backwards compatibility with existing app installations.
+* Relying Party (RP) can resolve user attributes from the Access by calling the User Info API or requesting an ID-Token. Both means are defined by OpenID Connect (OIDC) standard. The returned set of identity related claims couldn't be modified, extended or filtered (other than by using scope). Introduced User Details Customization Web Hook that serve these purposes.
+* Whenever the user device domain state changes, which could be caused by a new user device registration, deregistration, user logging in with the device, or mobile authentication enrollment changes, Access will publish a corresponding event to the event bus notifying all interested parties about the change.   
```

Comment:
Can we link to documentation of OneEx or is it not there yet?

Reply:
I don't think the OneEx docs are available publicly yet.